packages/pluggableWidgets/rich-text-web/package.json

```diff
@@ -14,39 +14,42 @@
   },
   "license": "Apache-2.0",
   "marketplace": {
-    "minimumMXVersion": "7.13.1",
+    "minimumMXVersion": "9.6.0",
     "marketplaceId": 74889
   },
+  "packagePath": "com.mendix.widget.custom",
   "scripts": {
-    "start": "utils-react-widgets start",
-    "dev": "utils-react-widgets dev",
-    "test": "npm run test:unit",
-    "test:unit": "jest --config ../../../scripts/test/jest.web.config.js",
+    "start": "cross-env MPKOUTPUT=RichText.mpk pluggable-widgets-tools start:server",
+    "dev": "cross-env MPKOUTPUT=RichText.mpk pluggable-widgets-tools start:web",
+    "build": "cross-env MPKOUTPUT=RichText.mpk pluggable-widgets-tools build:web",
+    "format": "pluggable-widgets-tools format",
+    "lint": "eslint --config ../../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
+    "test": "pluggable-widgets-tools test:unit:web",
     "pretest:e2e": "npm run release && node ../../../scripts/test/updateAtlas.js --latest-atlas",
     "test:e2e": "pluggable-widgets-tools test:e2e:web",
     "test:e2e:dev": "pluggable-widgets-tools test:e2e:web:dev",
-    "lint": "eslint --config ../../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
-    "lint:fix": "npm run lint -- --fix",
-    "build": "utils-react-widgets build",
-    "release": "utils-react-widgets release",
+    "release": "cross-env MPKOUTPUT=RichText.mpk pluggable-widgets-tools release:web",
     "release:marketplace": "node ../../../scripts/release/marketplaceRelease.js"
   },
   "config": {
     "mendixHost": "http://localhost:8080",
     "developmentPort": 3000
   },
   "devDependencies": {
-    "@mendix/custom-widgets-utils-internal": "0.0.1",
+    "@mendix/piw-utils-internal": "^1.0.0",
     "@mendix/pluggable-widgets-tools": ">=8.9.2",
     "@types/classnames": "^2.2.6",
-    "@types/quill": "^0.0.31",
     "@types/sanitize-html": "^1.18.2",
+    "@types/ckeditor4": "^4.16.1",
+    "@rollup/plugin-json": "^4.1.0",
+    "cross-env": "^7.0.2",
     "eslint": "^7.20.0",
-    "jest": "^26.6.1"
+    "through2": "^4.0.2"
   },
   "dependencies": {
     "classnames": "^2.2.6",
-    "quill": "^1.3.6",
-    "sanitize-html": "^2.0.0"
+    "sanitize-html": "^2.6.0",
+    "ckeditor4-react": "^2.1.0",
+    "ckeditor4": "^4.17.1"
```

Comment:
I know there was a reason, but I don't remember it clearly: 
- Why do we need both `ckeditor` and `ckeditor-react`?
- What is `ckeditor` used for and what is `ckeditor-react` used for?

Reply:
- `ckeditor` - the editor code itself (we going to copy bundle from this package + plugins)
- `ckeditor-react` - the react wrapper that actually manage `CKEditor` instance lifecycle

Reply:
Isn't ckeditor4 a dependency of ckeditor4-react?

Reply:
No
```sh
❯ npm info ckeditor4-react dependencies
{ 'ckeditor4-integrations-common': '^1.0.0', 'prop-types': '^15.7.2' }
```

Reply:
🤯

Reply:
Then I am resolving this discussion